arkouda/strings.py

```diff
@@ -1227,20 +1259,21 @@ def peel(
                 )
         if times < 1:
             raise ValueError("times must be >= 1")
-        cmd = "segmentedPeel"
-        args = "{} {} {} {} {} {} {} {} {} {}".format(
-            "peel",
-            self.objtype,
-            self.entry.name,
-            "str",
-            NUMBER_FORMAT_STRINGS["int64"].format(times),
-            NUMBER_FORMAT_STRINGS["bool"].format(includeDelimiter),
-            NUMBER_FORMAT_STRINGS["bool"].format(keepPartial),
-            NUMBER_FORMAT_STRINGS["bool"].format(not fromRight),
-            NUMBER_FORMAT_STRINGS["bool"].format(regex),
-            json.dumps([delimiter]),
+        repMsg = generic_msg(
+            cmd="segmentedPeel",
+            args={
+                "subcmd": "peel",
+                "objType": self.objtype,
+                "obj": self.entry,
+                "valType": "str",
+                "val": NUMBER_FORMAT_STRINGS["int64"].format(times),
```

Comment:
I know it was already `val` in the chapel code, but I think we should rename this to `times`. It immediately gets turned into `times` and something else becomes `val`, so I think making the name change sooner is clearer

Reply:
Yeah, good idea.

Reply:
Updated to `times`